lib/shared/bucketing-assembly-script/assembly/tsconfig.json

```diff
@@ -2,5 +2,8 @@
   "extends": "../../../../node_modules/assemblyscript/std/assembly.json",
   "include": [
     "./**/*.ts"
-  ]
+  ],
+  "compilerOptions": {
+    "strictPropertyInitialization": false
```

Comment:
this gets rid of errors from defining classes that only have properties and no constructor, which is valid Assemblyscript but was getting type errors..